test/parser/samples/unusual-identifier/input.html

```diff
@@ -0,0 +1,3 @@
+{{#each things as 𐊧}}
+	<p>𐊧</p>
```

Comment:
Did you intend for this to be `<p>{{𐊧}}</p>`?

Reply:
yep! fixed